.travis.yml

```diff
@@ -15,12 +17,17 @@ before_install:
   # The next couple lines fix a crash with multiprocessing on Travis and are not specific to using Miniconda
   - sudo rm -rf /dev/shm
   - sudo ln -s /run/shm /dev/shm
+  - git clone https://github.com/anderspitman/tidepool-docker
+  - tidepool-docker/build_containers tidepool-docker anderspitman
```

Comment:
@jh-bate currently yes. It simply clones each github repo. I designed the containers to mimic the behavior from the scripts in the tools repo as closely as possible. I could pin revisions without too much trouble if that is desired.